### PR TITLE
add default helm Ingress template

### PR DESCRIPTION
**Pull request by tomaszmichalowski**
Add default Ingress from helm default template.
Solves #48

### CODE REVIEW

File: charts/flipt/templates/service.yaml [hunk 1]
```diff
@@ -1,10 +1,33 @@
+{{- if .Values.service.enabled }}
```
Comment (GeorgeMac) [inline charts/flipt/templates/service.yaml:1]: I think this change is in your other PR, is that right?

**Review by GeorgeMac — CHANGES_REQUESTED** [review]
Thanks for the contribution (and associated issue)! This is indeed a gap in the templates.

I think you just need to bump the version. Maybe a patch will suffice, since this is a feature in the values which is not being acknowledged.
Also, am I right in thinking the changes to service.yaml in this PR is a duplicate of your other PR?

File: charts/flipt/templates/service.yaml (hunk 1 above)
Comment (tomaszmichalowski) [reply charts/flipt/templates/service.yaml]: I've removed this commit from that PR.

**Comment by GeorgeMac** [comment]
Hey @tomaszmichalowski I think you just need to merge in master or rebase 👍 
I think the chart version is conflicting since we merged in the other PR.

**Comment by tomaszmichalowski** [comment]
Main branch has beed merged to PR

**Review by GeorgeMac — APPROVED** [review]

**Comment by GeorgeMac** [comment]
Thanks @tomaszmichalowski 🙏

**Pull request merged**